nomos-services/data-availability/src/auth/mod.rs

```diff
@@ -0,0 +1,11 @@
+pub mod mock;
+
+pub trait Signer {
+    fn sign(&self, message: &[u8]) -> Vec<u8>;
+}
+
+pub trait DaAuth {
+    type Settings: Clone;
+
+    fn new(settings: Self::Settings) -> Self;
+}
```

Comment:
I don't get the main purpose of `DaAuth`

Reply:
Some parts of previous verification proposal are still in this PR. As discussed, the verification will be defined by the certificate implementation and only performed in the da mempool. `DaAuth` and other structures will be removed. I still need to add more changes to this, will ping reviewers once the draft is converted to a proper PR.